### DISCUSSION TITLE
Model 2 parameters-CL

### DISCUSSION THREAD

**Discussion by Mannny-PK** [LAPKB/Pmetrics#86] [Q&A]
Hello, about the parameters of the two-compartment pharmacokinetic model, I would like to ask a question: Does the CL parameter represent the central compartment clearance rate? How is it calculated?

my parameter settings
#Primary
Ke, 0.01,0.5
KCP, 0.1,10
KPC, 0.1,10
V, 5,60
Vp, 10,120
CL, 1,10

#Cov
WT
SCR
ALB
ASIAN
AGE
GENDER

#Out
Y(1) = X(1)/V

#Err
G=5
0.1, 0.15, 0, 0

Model results
![屏幕截图 2022-08-04 153822](https://user-images.githubusercontent.com/109020077/182791382-5e74129c-cb5e-4c34-8381-a58aaffc0f1e.png)

 Is this formula correct?  CL=ke*V

**Comment by mnneely** [comment]
CL is ignored the way you have it. To parameterize a model with CL please see the next to last example in our model library at http://www.lapk.org/ModelLib.php. Pmetrics first looks for a #DIFF block. Finding none, it then looks for key parameters in the #PRI or #SEC block named KE, V, KCP, KPC, KA. It must always find KE and V to use pre-specified model equations. KA, KCP, KPC all define more complex models, but with algebraic solutions. This avoids the need to remember things like ADVAN or TRANS used in other software. If you want to use CL in the #PRI block instead of KE, that’s fine, but let Pmetrics know in the #SEC block that KE = CL/V. Similar for Q and Vp.

On Aug 4, 2022, at 12:43 AM, Mannny-PK ***@***.******@***.***>> wrote:



Hello, about the parameters of the two-compartment pharmacokinetic model, I would like to ask a question: Does the CL parameter represent the central compartment clearance rate? How is it calculated?

my parameter settings
#Primary
Ke, 0.01,0.5
KCP, 0.1,10
KPC, 0.1,10
V, 5,60
Vp, 10,120
CL, 1,10

#Cov
WT
SCR
ALB
ASIAN
AGE
GENDER

#Out
Y(1) = X(1)/V

#Err
G=5
0.1, 0.15, 0, 0

Model results
[屏幕截图 2022-08-04 153822]<https://urldefense.com/v3/__https://user-images.githubusercontent.com/109020077/182791382-5e74129c-cb5e-4c34-8381-a58aaffc0f1e.png__;!!LIr3w8kk_Xxm!uonut06dI7lJf2UPC9FBliM_Zh5wxju6Z7y7GUGJh8kqzSdkIpdLNC7Fq_EHiFW-8gw1nRycfi2_eFIXhN8faEM$>

Is this formula correct? CL=ke*V

**Reply by Mannny-PK** [reply]
Thanks! I would like to ask another question: if the model shows that the convergence is False, does it mean that the model has failed?
![屏幕截图 2022-08-12 163445](https://user-images.githubusercontent.com/109020077/184317140-944abfa9-d306-4ded-bf6f-22e165086b9c.png)

**Comment by mnneely** [comment]
Nope. I just means the Log Likelihood is still changing more than the convergence criterion. Try running more cycles. However, I often only run 100 cycles (the default) early in model exploration and just try for convergence on my most promising model. NPAG comes very close to convergence usually within 10 cycles, so even if actual convergence occurs way beyond 100, the difference in support points isn’t usually much.
______________________________________


On Aug 12, 2022, at 1:39 AM, Mannny-PK ***@***.******@***.***>> wrote:



Thanks! I would like to ask another question: if the model shows that the convergence is False, does it mean that the model has failed?
[屏幕截图 2022-08-12 163445]<https://urldefense.com/v3/__https://user-images.githubusercontent.com/109020077/184317140-944abfa9-d306-4ded-bf6f-22e165086b9c.png__;!!LIr3w8kk_Xxm!rJhcD0TMf2_MfgD-E6nm2AP_7mdVtdkCOH-1YM8B1Znv7sdkgMm3cd1wJp_xtMYmLLlJFkTz6uvWFyn1odpSpLE$>

**Reply by Mannny-PK** [reply]
Thank you very much for your answer, I see what you mean.
